include/openthread/platform/crypto.h

```diff
@@ -610,6 +615,20 @@ otError otPlatCryptoEcdsaGenerateKey(otPlatCryptoEcdsaKeyPair *aKeyPair);
  */
 otError otPlatCryptoEcdsaGetPublicKey(const otPlatCryptoEcdsaKeyPair *aKeyPair, otPlatCryptoEcdsaPublicKey *aPublicKey);
 
+/**
+ * Import the provided key-pair into PSA.
+ *
+ * @param[in]  aKeyPair           A pointer to an ECDSA key-pair structure where the key-pair needs to be stored.
+ * @param[in]  aDerBytes          A pointer to an ECDSA key-pair in DER format.
+ * @param[in]  aDerLength         Length of the key-pair to be stored.
+ *
+ * @retval OT_ERROR_NONE          Public key was retrieved successfully, and @p aBuffer is updated.
+ * @retval OT_ERROR_PARSE         The key-pair DER format could not be parsed (invalid format).
+ * @retval OT_ERROR_INVALID_ARGS  The @p aContext is NULL.
+ *
+ */
+otError otPlatCryptoImportKeyPair(const otPlatCryptoEcdsaKeyPair *aKeyPair, uint8_t *aDerBytes, uint8_t aDerLength);
```

Comment:
If this function is supposed to save the `aDerByte` in PSA then we should just pass the `keyref`

```
otError otPlatCryptoImportKeyPair(uint16_t aKeyRef, const uint8_t *aDerBytes, uint8_t aDerLength);
```

Reply:
The plan was to have a single API work for importing both literal key and a keyref, but it seems to be cleaner just to introduce a new API just for PSA.